docs/notebooks/analysis/nullpoint.ipynb

```diff
@@ -0,0 +1,213 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2474e8ad-eaa6-4054-a3d3-dc0d8f21e10b",
+   "metadata": {},
+   "source": [
+    "# Null Point Finder\n",
+    "\n",
+    "[Null Point Finder]: ../../api_static/plasmapy.analysis.nullpoint.rst\n",
+    "\n",
+    "The [null point finder] is functionality that is designed to find and analyze 3D magnetic null point locations and structures using a trilinear method as described in [Haynes et al. (2007)](https://aip.scitation.org/doi/abs/10.1063/1.2756751)\n",
+    "In this notebook we will briefly cover how the [null point finder] utilizes the trilinear method in order to locate and classify the structures of magnetic null points. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9b3731e3-1936-4b26-8a05-2130669fd074",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "from pathlib import Path\n",
+    "\n",
+    "from plasmapy.analysis import nullpoint\n",
+    "\n",
+    "plt.rcParams[\"figure.figsize\"] = [10.5, 0.56 * 10.5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eec0e81c-bbf8-4e01-b121-114401996f7c",
+   "metadata": {},
+   "source": [
+    "## Contents:\n",
+    "\n",
+    "1. [How Null Point Finder works](#How-Null-Point-Finder-works)\n",
+    "    1. [Locating a null point](#Locating-a-null-point)\n",
+    "    1. [Classifying a null point](#Classifying-a-null-point)\n",
+    "1. [Running through examples](#Running_through-examples)\n",
+    "    1. [Uniform regular grid with a model function](#Uniform-regular-grid-with-a-model-function)\n",
+    "    1. [Arbitary Regular Grid](#Arbitary-regular-grid)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "60702e11-905b-43b3-85ab-dc3f85bc9bab",
+   "metadata": {},
+   "source": [
+    "## How Null Point Finder works\n",
+    "Null point finder provides two functions that can locate and classify null points in a magnetic field. The first function is [**uniform_null_point_find()**](../../api/plasmapy.analysis.nullpoint.uniform_null_point_find.rst)  and the second one is [**null_point_find()**](../../api/plasmapy.analysis.nullpoint.null_point_find.rst). As the names suggest, `uniform_null_point_find()` is used to locate and classify the magnetic null points of magnetic field located within a regular grid with uniform spacing in each dimension. It requires the user to provide the spacing between grid points in each of the three dimensions in addition to the minimum and maximum coordinate in each dimension. Moreover, it requires the user to provide a function which generates the vector values at a given point. `uniform_null_point_find()` is useful for when the user knows of such a function that accurately models the magnetic field vector at each point in space. \n",
```

Comment:
```suggestion
    "Null point finder provides two functions that can locate and classify null points in a magnetic field. The first function is [uniform_null_point_find()](../../api/plasmapy.analysis.nullpoint.uniform_null_point_find.rst)  and the second one is [null_point_find()](../../api/plasmapy.analysis.nullpoint.null_point_find.rst). As the names suggest, `uniform_null_point_find()` is used to locate and classify the magnetic null points of magnetic field located within a regular grid with uniform spacing in each dimension. It requires the user to provide the spacing between grid points in each of the three dimensions in addition to the minimum and maximum coordinate in each dimension. Moreover, it requires the user to provide a function which generates the vector values at a given point. `uniform_null_point_find()` is useful for when the user knows of such a function that accurately models the magnetic field vector at each point in space. \n",
```
...since the formatting didn't get interpreted as bold in the doc build.